README.md

```diff
@@ -3,8 +3,11 @@
 How to start the dropwizard-example application
 ---
 
+1. Make sure both [`dropwizard-logstash`](https://github.com/alphagov/dropwizard-logstash) and [`dropwizardPrometheus`](https://github.com/alphagov/gds_metrics_dropwizard) 
+are at the right version in [`external-dependencies.gradle`](external-dependencies.gradle). If they are not published on Maven Central, you may want to publish them to
+your Maven local repository.
```

Comment:
`dropwizard-logstash` is published to Bintray. we shouldn't be making people publish it to maven local 😞